examples/Remarketing/UploadOfflineConversion.cs

```diff
@@ -140,12 +167,37 @@ public static void Main(string[] args)
             ClickConversion clickConversion = new ClickConversion()
             {
                 ConversionAction = ResourceNames.ConversionAction(customerId, conversionActionId),
-                Gclid = gclid,
                 ConversionValue = conversionValue,
                 ConversionDateTime = conversionTime,
                 CurrencyCode = "USD"
             };
 
+            // Verifies that exactly one of gclid, gbraid, and wbraid is specified, as required.
+            // See https://developers.google.com/google-ads/api/docs/conversions/upload-clicks
+            // for details.
+            string[] ids = { gclid, gbraid, wbraid };
+            int idCount = ids.Where(id => !string.IsNullOrEmpty(id)).Count();
+
+            if (idCount != 1)
+            {
+                throw new ArgumentException($"Exactly 1 of gclid, gbraid, or wbraid is " +
+                    $"required, but {idCount} ID values were provided");
+            }
+
+            // Sets the single specified ID field.
+            if (!string.IsNullOrEmpty(gclid))
+            {
+                clickConversion.Gclid = gclid;
+            }
+            if (!string.IsNullOrEmpty(wbraid))
```

Comment:
I realize it won't change the behavior since you already check the count, but would it be clearer that you're only going to set one of them if lines 192 and 196 were `else if` statements?

Reply:
Done.